code/unclassified/src/Max_of_array/Max_of_array.cpp

```diff
@@ -0,0 +1,27 @@
+ #include <iostream>
+using namespace std;
+
+int main() {
+	//the largest number in an array;
+	//declaring an array
+	int n;
+	cout<<"enter the size of the array"<<endl; //getting the number of elements of the array
+	cin>>n;//taking input
+	int arr[n]; //creating array arr
```

Comment:
This line is not standard compliant. You can use a dynamic array or std::vector.